src/ops/nn.jl

```diff
@@ -1,119 +1,52 @@
 module nn
 
-export
-conv2d,
-max_pool,
-zero_state,
-output_size,
-state_size,
-rnn,
-dynamic_rnn,
-dropout,
-relu,
-relu6,
-elu,
-softplus,
-softsign,
-softmax,
-sigmoid,
-tanh,
-state_saving_rnn,
-bidirectional_rnn,
-sigmoid_cross_entropy_with_logits,
-sparse_softmax_cross_entropy_with_logits,
-softmax_cross_entropy_with_logits,
-log_softmax,
-embedding_lookup,
-top_k,
-in_top_k,
-l2_loss,
-log_poisson_loss,
-nce_loss,
-sampled_softmax_loss,
-batch_normalization,
-seq2seq,
-conv2d_transpose
-
 using Compat
-import ..TensorFlow: Operation, NodeDescription, get_def_graph, capitalize, add_input, Port, get_name, set_attr_list, get_shape, variable_scope, shape, random_uniform, AbstractTensor, Tensor, reduce_sum, @not_implemented, with_op_name, @op, @tf
 import TensorFlow
 const tf = TensorFlow
+import .tf: Ops, @op, @not_implemented, @tf
```

Comment:
What is the reason for cutting down on imports for parent module?

I was thinking yesterday we should do the opposite and just say `using ..TensorFlow`